### PR TITLE
Make Pay Button a plugin

### LINKED ISSUE #3701: `Pay Button` to become a system plugin
As mentioned in the Mattermost, would be a nice improvement and consolidation to clean up the "Payments" section.

- [ ] Pay Button plug-in
- [ ] Move current position in left sidebar

If this is not feasible, or there's something I'm overlooking, please feel free to close the issue.

### PR DESCRIPTION
**Pull request by dennisreimann**
Closes #3701.

### CODE REVIEW

**Comment by dstrukt** [comment]
Other than the known out-of-scope issue w.r.t the warning (https://github.com/btcpayserver/btcpayserver/issues/3535).. 

LGTM, tACK!

**Review by dstrukt — APPROVED** [review]

**Review by pavlenex — COMMENTED** [review]
LGTM, just a minor nit, also made sure plugin description mentions that it should be used for tips and donations, which gives more context to what we should tackle in https://github.com/btcpayserver/btcpayserver/issues/3535

File: BTCPayServer/Plugins/PayButton/PayButtonPlugin.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+using BTCPayServer.Abstractions.Services;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace BTCPayServer.Plugins.PayButton
+{
+    public class PayButtonPlugin : BaseBTCPayServerPlugin
+    {
+        public override string Identifier => "BTCPayServer.Plugins.PayButton";
+        public override string Name => "PayButton";
+        public override string Description => "Easily-embeddable HTML and highly-customizable payment button.";
```
[the remark this answers is not in the document]
Comment (dennisreimann) [reply BTCPayServer/Plugins/PayButton/PayButtonPlugin.cs]: Thanks, applied!

**Comment by NicolasDorier** [comment]
Any update to do to the doc?

**Comment by dennisreimann** [comment]
@NicolasDorier thanks for merging! The docs addition is tracked in btcpayserver/btcpayserver-doc#754

**Pull request merged**